### ISSUE TITLE
Map not loading on upload page in Safari 10.1.2 or earlier

### ISSUE THREAD

**Issue by tiwane** [inaturalist/inaturalist#2012]
When using Safari 10.1.2 or earlier, the map does not appear in the Location pop-up modal on the upload page. @kueda mentioned that it's likely "failing flexbox support: https://caniuse.com/#search=flex" Probably worth a fix as Version 10 isn't too old.

**Comment by pleary** [comment]
I don't think so. If I recall that was needed for asset precompilation and/or webpack. So I say let's remove it, and as long as deploys work we should be fine

**Comment by kueda** [comment]
FWIW, the flexbox stuff applies to broken header and location chooser layouts Tony observed in Safari 8. I'm not really sure how much I can do to address the map issue without a working version of Safari 10 in front of me.
